TextBox.py

```diff
@@ -42,14 +48,14 @@ def replace_bytes(match: re.Match) -> str:
         text = hex_string_regex.sub(replace_bytes, text)
 
     # Parse the text into a list of control codes.
-    text_codes = Messages.parse_control_codes(text)
+    text_codes = Messages.parse_control_codes(text, lang)
 
     # Existing line/box break codes to strip.
     strip_codes = []
     if strip_existing_boxes:
-        strip_codes.append(0x04)
+        strip_codes.append([0x81A5,0x04][lang])
```

Comment:
Commas should have a space after them (applies to many other places in this file)